### ISSUE TITLE
Refinement of Related Signs view

### ISSUE THREAD

**Issue by ocrasborn** [Signbank/NGT-signbank#265]
* After #110 
* Add another category 'Morphological pairs'
* Move all minimal pairs to this block of which the contrasting phonological feature (the one being the sole difference between two related signs) is listed in the morphology table as the phonological property of one of the morphemes

**Comment by ocrasborn** [comment]
Update: let's postpone this for a while until Els/Inge have finished their data collection (2018).

**Comment by susanodd** [comment]
This issue can probably be closed as it seems to have been implemented, albeit in other issues.
